spec/controllers/reducers_controller_spec.rb

```diff
@@ -96,6 +96,16 @@
         expect(workflow.reducers.first.filters['extractor_keys']).to eq(['test'])
       end
 
+      it 'saves extractor_keys as an array' do
+        nested_reducer_params[:extractor_keys] = 'test'
```

Comment:
Thats weird.  I would have expected if we set extractor keys input to a string that it would keep as string. Maybe you mean `nested_reducer_params[:filters][:extractor_keys]` ?